sdk/core/azure-core/tests/test_streaming.py

```diff
@@ -43,6 +44,17 @@ def test_decompress_plain_no_header(http_request):
     decoded = content.decode('utf-8')
     assert decoded == "test"
 
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
+def test_compress_plain_no_header_offline(port, http_request):
+    # thanks to Daisy Cisneros for this test!
+    # expect plain text
+    request = http_request(method="GET", url="http://localhost:{}/streams/string".format(port))
+    with RequestsTransport() as sender:
+        response = sender.send(request)
+        response.raise_for_status()
+        assert response.text() == "test"
```

Comment:
thanks @DaisyCisneros this looks really great! Can you also add to this test by trying to stream download the response? You can follow lines 66 -> 71 in this file

Reply:
also just to be clear, you don't have to switch to `pipeline.run()`, the transport.send() method takes in kwarg `stream` as well, so just make sure you pass in `stream=True`

Reply:
Thank you! Lines 66-71 were a very helpful guide to making the response stream download. I made sure to pass in stream=True as well 😊. These changes are reflected in the most recent commit!